scripts/templates/init-defaultworker.template

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
 cd /hydroshare
-usermod -u HS_SERVICE_UID hydro-service
-groupmod -g HS_SERVICE_GID storage-hydro
```

Comment:
are there not cleaner user management routines?

Reply:
Same above